src/main/scala/uk/gov/nationalarchives/tdr/api/metadatainputvalidation/MetadataInputValidator.scala

```diff
@@ -0,0 +1,32 @@
+package uk.gov.nationalarchives.tdr.api.metadatainputvalidation
```

Comment:
Can we rename the package to `uk.gov.nationalarchives.tdr.api.validator` and move all the files related to validation to the `validator` package?

AuthorisationTags.scala
ValidationAuthoriser.scala
ConsignmentStateTags.scala
ConsignmentStateValidator.scala
MetadataInputTags.scala
MetadataInputValidator

Reply:
I'd rather leave them in separate packages for now. Each package represents a different type check, eg authorisation, consignment state, metadata input. Its something we can reassess if we need to add/make further adjustments to the validators.